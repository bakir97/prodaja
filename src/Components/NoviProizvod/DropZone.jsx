import React from "react";
import Dropzone from "react-dropzone";
import { Grid } from "@material-ui/core";
const style = {
  dropzone: {
    border: "1px solid rgba(0, 0, 0, 0.23)",
    maxWidth: 600,
    height: 113,
    width: "100%",
    margin: 10,
    borderRadius: 4
  },
  text: { color: "rgba(0, 0, 0, 0.5)", paddingLeft: 10 }
};
export default ({ drop }) => {
  return (
    <Grid item xs={12} container justify="center">
      <Dropzone onDrop={files => drop(files)} multiple style={style.dropzone}>
        <p style={style.text}>Prenesi slike ovdje ili pritisni za odabir</p>
      </Dropzone>
    </Grid>
  );
};
