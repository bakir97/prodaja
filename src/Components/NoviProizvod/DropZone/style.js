export default {
  dropzone: {
    border: "1px solid rgba(0, 0, 0, 0.23)",
    maxWidth: 600,
    height: 113,
    width: "100%",
    margin: 10,
    borderRadius: 4
  },
  text: { color: "rgba(0, 0, 0, 0.5)", paddingLeft: 10 }
};
