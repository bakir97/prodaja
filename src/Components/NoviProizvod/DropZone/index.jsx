import React from "react";
import Dropzone from "react-dropzone";
import { Grid } from "@material-ui/core";
import style from "./style";
export default ({ drop }) => {
  return (
    <Grid item xs={12} container justify="center">
      <Dropzone onDrop={files => drop(files)} multiple style={style.dropzone}>
        <p style={style.text}>Prenesi slike ovdje ili pritisni za odabir</p>
      </Dropzone>
    </Grid>
  );
};
