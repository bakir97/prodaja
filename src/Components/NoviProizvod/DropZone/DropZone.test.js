import DropZone from "./index";
describe("DropZone", () => {
  const wrapper = shallow(<DropZone />);
  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
