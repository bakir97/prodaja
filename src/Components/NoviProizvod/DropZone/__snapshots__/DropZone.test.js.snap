// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropZone should render 1`] = `
<WithStyles(Grid)
  container={true}
  item={true}
  justify="center"
  xs={12}
>
  <n
    disableClick={false}
    disablePreview={false}
    disabled={false}
    getDataTransferItems={[Function]}
    inputProps={Object {}}
    maxSize={Infinity}
    minSize={0}
    multiple={true}
    onDrop={[Function]}
    preventDropOnDocument={true}
    style={
      Object {
        "border": "1px solid rgba(0, 0, 0, 0.23)",
        "borderRadius": 4,
        "height": 113,
        "margin": 10,
        "maxWidth": 600,
        "width": "100%",
      }
    }
  >
    <p
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.5)",
          "paddingLeft": 10,
        }
      }
    >
      Prenesi slike ovdje ili pritisni za odabir
    </p>
  </n>
</WithStyles(Grid)>
`;
